core/src/main/java/com/linecorp/armeria/client/circuitbreaker/MetricCollectingCircuitBreakerListener.java

```diff
@@ -33,29 +33,29 @@
  * <table summary="metrics that will be generated by this class">
  * <tr><th>metric name</th><th>description</th></tr>
  *
- * <tr><td>{@code <name>_requests{name="<circuit breaker name>",result="success"}}</td>
+ * <tr><td>{@code <name>.requests{name="<circuit breaker name>",result="success"}}</td>
  * <td>The number of successful requests in the counter time window.</td></tr>
  *
- * <tr><td>{@code <name>_requests{name="<circuit breaker name>",result="failure"}}</td>
+ * <tr><td>{@code <name>.requests{name="<circuit breaker name>",result="failure"}}</td>
  * <td>The number of failed requests in the counter time window.</td></tr>
  *
- * <tr><td>{@code <name>_transitions{name="<circuit breaker name>",state="CLOSED"}}</td>
+ * <tr><td>{@code <name>.transitions{name="<circuit breaker name>",state="CLOSED"}}</td>
  * <td>The number of circuit breaker state transitions to {@link CircuitState#CLOSED}.</td></tr>
  *
- * <tr><td>{@code <name>_transitions{name="<circuit breaker name>",state="OPEN"}}</td>
+ * <tr><td>{@code <name>.transitions{name="<circuit breaker name>",state="OPEN"}}</td>
  * <td>The number of circuit breaker state transitions to {@link CircuitState#OPEN}.</td></tr>
  *
- * <tr><td>{@code <name>_transitions{name="<circuit breaker name>",state="HALF_OPEN"}}</td>
+ * <tr><td>{@code <name>.transitions{name="<circuit breaker name>",state="HALF_OPEN"}}</td>
  * <td>The number of circuit breaker state transitions to {@link CircuitState#HALF_OPEN}.</td></tr>
  *
- * <tr><td>{@code <name>_rejected_requests{name="<circuit breaker name>"}}</td>
+ * <tr><td>{@code <name>.rejectedRequests{name="<circuit breaker name>"}}</td>
  * <td>The number of requests rejected by the circuit breaker.</td></tr>
  *
  * </table>
  */
 public final class MetricCollectingCircuitBreakerListener implements CircuitBreakerListener {
 
-    private static final String DEFAULT_NAME = "armeria_client_circuit_breaker";
+    private static final String DEFAULT_NAME = "armeria.client.circuitBreaker";
```

Comment:
I think we should distinguish between ``tag``name and ``metric``name. 
maybe ``METRIC_NAME = ...``